_test/test_data_loaders/test_rundata.m

```diff
@@ -345,6 +347,33 @@ function test_serialization_crystal(this)
             
             assertEqual(run,run1);
         end
+        %
+        function test_serialize_on_file(obj)
+            %
+            run=rundata(f_name(obj,'MAP11014.nxspe'));
+            db = run.serialize();
+            runr = rundata.deserialize(db);
+            
+            assertEqual(run,runr);
+        end
+        %
+        function test_serialize_in_memory(obj)
```

Comment:
what is this test testing? The previous test has shown `run === runr`, this seems to test that `run === runr` with some fields removed

Reply:
it checks that serialization/deserialization work